django/db/backends/oracle/base.py

```diff
@@ -387,8 +388,7 @@ def __init__(self, connection):
         self.cursor = connection.cursor()
         # Necessary to retrieve decimal values without rounding error.
         self.cursor.numbersAsStrings = True
-        # Default arraysize of 1 is highly sub-optimal.
-        self.cursor.arraysize = 100
+        self.cursor.arraysize = constants.GET_ITERATOR_CHUNK_SIZE
```

Comment:
I think we can leave the code as is since it can be removed after support for cx_Oracle < 5.3 is removed (see https://github.com/oracle/python-cx_Oracle/commit/a56f7056). I updated the comment in eca6f91be2832208703d91e541e6f0b0addabfe0.